app/models/user.rb

```diff
@@ -293,8 +223,12 @@ class User < ApplicationRecord
   validate :password_matches_confirmation, if: :encrypted_password_changed?
 
   alias_attribute :public_reactions_count, :reactions_count
+
+  # [@msarit] to remove this once we've moved the related code
```

Comment:
@Ridhwana @rhymes @citizen428 I'm not clear on these two comments; it may be coding fatigue on my part 😓